extensions/NV/EGL_NV_context_priority_realtime.txt

```diff
@@ -0,0 +1,194 @@
+Name
+
+    NV_context_priority_realtime
+
+Name Strings
+
+    EGL_NV_context_priority_realtime
+
+Contributors
+
+    Sandeep Shinde
+    Kirill Artamonov
+    Sami Kiminki
+    Donghan Ryu
+    Daniel Koch
+    Mark Kilgard
+
+Contacts
+
+    Sandeep Shinde, NVIDIA (sashinde 'at' nvidia 'dot' com)
+
+Status
+
+    Complete
+
+Version
+
+    Version 4 - November 21, 2017
+
+Number
+
+    EGL Extension #124
+
+Dependencies
+
+    Requires EGL 1.0.
+
+    Requires EGL_IMG_context_priority
+
+    This extension is written against the wording of the EGL 1.5
+    Specification - August 27, 2014 (but may be implemented against earlier
+    versions).
+
+Overview
+
+    This extension allows an EGLContext to be created with one extra
+    priority level in addition to three priority levels that are part of
+    EGL_IMG_context_priority extension.
+
+    This new level has extra privileges that are not available to other three
+    levels. Some of the privileges may include:
+    - Allow realtime priority to only few contexts
+    - Allow realtime priority only to trusted applications
+    - Make sure realtime priority contexts are executed immediately
+    - Preempt any current context running on GPU on submission of
+      commands for realtime context
+
+    The mechanism for determining which EGL context is allowed to use this
+    priority level is platform dependent.
+
+New Types
+
+    None
+
+New Procedures and Functions
+
+    None
+
+New Tokens
+
+    New attribute values accepted in the <attrib_list> argument
+    of eglCreateContext:
```

Comment:
I would specify the attribute this applies to:
> New attribute values accepted for the EGL_CONTEXT_PRIORITY_LEVEL_IMG attribute in the <attrib_list> argument of eglCreateContext

Reply:
Right, I'll update the spec.